holoviews/tests/core/test_callable.py

```diff
@@ -66,6 +69,9 @@ def test_callable_class_name(self):
     def test_callable_class_call_method(self):
         self.assertEqual(Callable(CallableClass().__call__).name, 'CallableClass')
 
+    def test_callable_instance_method(self):
+        self.assertEqual(Callable(CallableClass().someinstancemethod).name, 'CallableClass.someinstancemethod')
```

Comment:
I know the other tests use `self.assertEqual`, but I want to transition away from using the unittest framework to pytest. So can you change all of your added tests to `assert a == b`?